keras/utils/vis_utils.py

```diff
@@ -13,7 +13,13 @@
 
 
 def _check_pydot():
-    if not (pydot and pydot.find_graphviz()):
+    try:
+        if hasattr(pydot, 'find_graphviz'):
+            if not pydot.find_graphviz():
+                raise ImportError()
```

Comment:
This needs an error message explaining what went wrong and how to fix it (like below).